### PR TITLE
Tutorials for calculating IR added

### PR DESCRIPTION
**Pull request by WeiLiweili**
Tutorials for calculating IR added

### CODE REVIEW

File: src/mmath/linalg/MATRIX.cpp [hunk 1]
```diff
@@ -824,7 +824,7 @@ void MATRIX::show_matrix(char * Output_File)
    if(ob.is_open())
    {
    int k=0;
-   ob<<endl;
+   //ob<<endl;
```
Comment (alexvakimov) [inline src/mmath/linalg/MATRIX.cpp]: why was it necessary? this is a convention. Please return

File: tests/study3/calculate_ir.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    from cyglibra_core import *
+elif sys.platform=="linux" or sys.platform=="linux2":
+    from liblibra_core import *
+
+
+from libra_py import *
+
+
+
+def get_Num_atoms(fname):
```
Comment (alexvakimov) [inline tests/study3/calculate_ir.py]: please document the input variables

File: tests/study3/calculate_ir.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+# get the number of the atoms from the trajectory file
+
+	with open(fname,'r') as fo:
+		Natom = int(fo.next())
+		
+	return Natom
+
+def read_3d_data(fname,num_atoms):
```
Comment (alexvakimov) [inline tests/study3/calculate_ir.py]: please document the input variables

File: tests/study3/calculate_ir.py [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+			for n in range(0,num_atoms):
+				line = fo.next()
+				coords[time_step][n]= line.split()[1:]
+				
+			time_step += 1
+			
+			
+	return (coords,time_step)
+
+def read_charge(fname):
```
Comment (alexvakimov) [inline tests/study3/calculate_ir.py]: please document the input variables

File: tests/study3/calculate_ir.py [hunk 5]
```diff
@@ -0,0 +1,160 @@
+#*********************************************************************************
```
Comment (alexvakimov) [inline tests/study3/calculate_ir.py]: in this code, please make sure you use 4 spaces per indentation level, also make sure you don't use Tabs instead of the whitespaces

File: src/mmath/linalg/MATRIX.cpp (hunk 1 above)
Comment (WeiLiweili) [inline src/mmath/linalg/MATRIX.cpp]: I add a small test in my git repo (https://github.com/WeiLiweili/test/tree/master/test-show_matrix)

When we want to do some i/o operation on the output file, if we don't comment that line, then line 20, a.py, will terminate with errors since first line is a blank line:
 ```
 File "a.py", line 20, in <module>
    print float(i.split())*2
TypeError: float() argument must be a string or a number
```
Maybe this is just needed conditionally.

Comment (alexvakimov) [inline src/mmath/linalg/MATRIX.cpp]: Then, you need to change a.py code such that it works with the format that is already assumed in Libra

Comment (WeiLiweili) [inline src/mmath/linalg/MATRIX.cpp]: thank you, I will make the changes.

Comment (alexvakimov) [inline src/mmath/linalg/MATRIX.cpp]: Please use "git checkout" to revert the changes made to MATRIX.cpp - this is a proper way to work with git (not just uncommenting your previous comment) - we don't need the history of unnecessary code modifications.
